pkgs/by-name/hm/hmat-oss/package.nix

```diff
@@ -3,38 +3,46 @@
   stdenv,
   fetchFromGitHub,
   cmake,
+  pkg-config,
   blas,
   lapack,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "hmat-oss";
   version = "1.10.0";
 
   src = fetchFromGitHub {
     owner = "jeromerobert";
     repo = "hmat-oss";
-    tag = version;
-    sha256 = "sha256-GnFlvZCEzSCcBVLjFWLe+AKXVA6UMs/gycrOJ2TBqrE=";
+    tag = finalAttrs.version;
+    hash = "sha256-GnFlvZCEzSCcBVLjFWLe+AKXVA6UMs/gycrOJ2TBqrE=";
   };
 
   cmakeFlags = [
     "-DHMAT_GIT_VERSION=OFF"
   ];
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs =
+    [
+      cmake
+    ]
+    ++ lib.optionals stdenv.hostPlatforms.idDarwin [
```

Comment:
is?

Reply:
Oh, not sure where that typo came from. fixed!